client/tests/integration/status_response.rs

```diff
@@ -0,0 +1,54 @@
+use std::str::FromStr as _;
+
+use eyre::Result;
+use iroha_client::data_model::prelude::*;
+use iroha_telemetry::metrics::Status;
+use parity_scale_codec::DecodeAll;
+use test_network::*;
+
+fn status_eq_excluding_uptime_and_queue(lhs: &Status, rhs: &Status) -> bool {
+    lhs.peers == rhs.peers
+        && lhs.blocks == rhs.blocks
+        && lhs.txs_accepted == rhs.txs_accepted
+        && lhs.txs_rejected == rhs.txs_rejected
+        && lhs.view_changes == rhs.view_changes
+}
+
+#[test]
+fn json_and_scale_statuses_equality() -> Result<()> {
+    let (_rt, network, client) = Network::start_test_with_runtime(1, Some(10_935));
```

Comment:
10_925 is occupied with `unstable_network_stable_after_add_and_after_remove_peer` which establish network of 4 peers, so 10_935 is overlapping.

Reply:
Changed. Also changed peers to 2.
I noticed that there are lots of tests that are currently ignored due to slowness. Can reduction of n_peers help with that, what do you think?